src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -370,19 +370,25 @@ public function makePathRelative($endPath, $startPath)
         if ('\\' === DIRECTORY_SEPARATOR) {
```

Comment:
Comment above needs to be updated. This block no longer just normalizes separators